site/content/docs/5.0/components/carousel.md

```diff
@@ -46,7 +46,7 @@ Here's a carousel with slides only. Note the presence of the `.d-block` and `.w-
 
 ### With controls
 
-Adding in the previous and next controls:
+Adding in the previous and next controls. We recommend using `<button>` elements, but you can also use `<a>` link elements–though in that case, you should also add `role="button"` attributes to the links.
```

Comment:
Can make this short and sweet IMO.

```suggestion
Adding in the previous and next controls. We recommend using `<button>` elements, but you can also use `<a>` elements with `role="button"`.
```

Reply:
that might be misleading, as it may suggest our code only supports `<a>` with `role="button"`, while in effect it doesn't really check for `role="button"`?

Reply:
@mdo would you be ok with the more verbose, but possibly clearer, wording?

Reply:
I think brevity and best practice should win out—if folks use anchor elements, they really should use `role="button"`, yes? If that's not the case, then yeah we can use the more verbose.

Reply:
*shrug* fairy snuff :)